### PR TITLE
Solaris: fix readdir for x64 OS.

### PR DESCRIPTION
**Pull request by nykytenko**
In x64 OS not present readdir64.

### CODE REVIEW

File: src/core/sys/posix/dirent.d [hunk 1]
```diff
@@ -180,14 +180,21 @@ else version (Solaris)
         char* dd_buf;
     }
 
-    static if (__USE_LARGEFILE64)
+    version (D_LP64)
     {
-        dirent* readdir64(DIR*);
-        alias readdir64 readdir;
+        dirent* readdir(DIR*);
```
Comment (DmitryOlshansky) [inline src/core/sys/posix/dirent.d]: add an alias for readdir64

**Comment by nykytenko** [comment]
For x64 Solaris kernel no need readdir64.

**Comment by MartinNowak** [comment]
Auto-merge toggled on

**Comment by nykytenko** [comment]
Done.

**Pull request merged**